index.bs

```diff
@@ -551,9 +556,12 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |issuedRequests| and |currentlyAvailableAuthenticators| be new [=ordered sets=].
 
-1. For each |authenticator| currently available on this platform, if
-    <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
+1. For each |authenticator| currently available on this platform:
+    1. If <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
+        |authenticator|'s attachment modality, continue to next step. Otherwise, do not [=set/append=] |authenticator|
```

Comment:
"do not X" is odd to see in a spec algorithm. Not-doing things is the default. ;)

Maybe `If <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present=] and its value does not match |authenticator|'s attachment modality, [=continue=].`